demos/object_detection_demo/python/README.md

```diff
@@ -39,6 +39,12 @@ Async API operates with a notion of the "Infer Request" that encapsulates the in
 
 > **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with the `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#general-conversion-parameters).
 
+## Model API
+
+The demo utilizes model wrappers, adapters and pipelines from [Python* Model API](../../common/python/openvino/model_zoo/model_api/README.md).
+
+The generalized interface of wrappers with its unified results representation provides the easy support of many different object detection model topologies in one demo.
```

Comment:
```suggestion
The generalized interface of wrappers with its unified results representation provides the support of multiple different object detection model topologies in one demo.
```
It's not up to you to judge if the support is easy. Same about the number of models